src/libstd/fs.rs

```diff
@@ -1724,6 +1734,23 @@ mod tests {
         assert!(canary.exists());
     }
 
+    // FIXME(#12795) depends on lstat to work on windows
+    #[cfg(not(windows))]
```

Comment:
So should I remove the comment but leave cfg(not(windows))? (so use symlink from os::unix)

I'm pretty sure tests for windows should be separate testing both symlink_file, and symlink_dir. And I'm unable to write them.